2023_VillaIslas_Science/2023_VillaIslas_Science.janno

```diff
@@ -0,0 +1,28 @@
+Poseidon_ID	Genetic_Sex	Group_Name	Country	Country_ISO	Location	Site	Latitude	Longitude	Date_Type	Date_BC_AD_Start	Date_BC_AD_Median	Date_BC_AD_Stop	Date_Note	MT_Haplogroup	Y_Haplogroup	Source_Tissue	Nr_Libraries	Library_Names	Capture_Type	UDG	Library_Built	Genotype_Ploidy	Data_Preparation_Pipeline_URL	Endogenous	Nr_SNPs	Damage	Contamination	Contamination_Err	Contamination_Meas	Contamination_Note	Genetic_Source_Accession_IDs	Publication	Eager_ID	Main_ID	RateErrX	RateErrY	RateX	RateY
+11R_R_b_MNT	M	Ranas;CentralMexico;SierraGorda	Mexico	MX	Querétaro	Ranas	20.92612	-99.56488	contextual	729	769	809	Dates are taken from Date (CE) in S6	D1m	Q1a2a1-L54	tooth	2	mtDNA-genome;Whole-genome	Shotgun	minus	ds	haploid	https://github.com/nf-core/eager/releases/tag/2.4.6	99.296225	8454	0.25554637284974846	n/a	n/a	n/a	Nr Snps (per library): 0;0. Estimate and error are weighted means of values per library. Libraries with fewer than 100 SNPs used in contamination estimation were excluded.	PRJEB51440;ERS10903714;ERR9576909;ERR9169797	2023VillaIslasScience	11R_R_b	11R_R_b	0.033425961437661	0.056720746903437	0.389404695593133	0.73410877311746
+2417C_TOL_a_MNT	F	Toluquilla;CentralMexico;SierraGorda	Mexico	MX	Querétaro	Toluquilla	20.88378	-99.53097	contextual	1100	1160	1220	Dates are taken from Date (CE) in S11	A2d	n/a	tooth	1	mtDNA-genome	Shotgun	minus	ds	haploid	https://github.com/nf-core/eager/releases/tag/2.4.6	98.932551	0	0.3336998350742166	n/a	n/a	n/a	Nr Snps (per library): 0. Estimate and error are weighted means of values per library. Libraries with fewer than 100 SNPs used in contamination estimation were excluded.	PRJEB51440;ERS10903719;ERR9576910	2023VillaIslasScience	2417C_TOL_a	2417C_TOL_a	0.0	0.0	0.0	0.0
```

Comment:
Aha - I got the misplaced info in the `Library_Names` column now. So for this and other samples only mtDNA was sequenced. As a consequence the number of covered SNPs is zero. I guess it was a conscious decision to include these samples anyway in the Poseidon package, right?